### ISSUE TITLE
docker-compose up --build issue

### ISSUE THREAD

**Issue by kaustubhshete** [screenshotbot/screenshotbot-oss#25]
I am getting following effor since I am behind corporate proxy:

Step 18/18 : CMD /opt/software/ccl/lx86cl64 -l launch.lisp -- --object-store /data/ --start-slynk --slynk-port 4005 --slynk-loopback-interface 0.0.0.0
 ---> Using cache
 ---> efdf4c10713d
Successfully built efdf4c10713d
Successfully tagged screenshotbot-oss_screenshotbot:latest
Starting screenshotbot-oss_screenshotbot_1 ... done
Attaching to screenshotbot-oss_screenshotbot_1
screenshotbot_1  | Compiling asdf..
screenshotbot_1  | To load "log4cl":
screenshotbot_1  |   Load 1 ASDF system:
screenshotbot_1  |     asdf
screenshotbot_1  |   Install 3 Quicklisp releases:
screenshotbot_1  |     alexandria bordeaux-threads log4cl
screenshotbot_1  | > Error: Network is unreachable (error #101) during attempt to connect to 13.33.60.26:80
screenshotbot_1  | > While executing: SOCKET-ERROR, in process listener(1).
screenshotbot_1  | > Type :GO to continue, :POP to abort, :R for a list of available restarts.
screenshotbot_1  | > If continued: Skip loading "launch.lisp"
screenshotbot_1  | > Type :? for other options.
screenshotbot_1  | 1 > screenshotbot-oss_screenshotbot_1 exited with code 0

**Comment by tdrhq** [comment]
Taking a look right now. I'm guessing it's a transient issue with Quicklisp, the package manager for CL

**Comment by tdrhq** [comment]
Oh, you're behind a corporate proxy. Do you have a proxy server that you can configure? I need to double check if Quicklisp will support the HTTP_PROXY out of the box, but most likely you'll need to some special configuring to make the proxy work.

**Comment by kaustubhshete** [comment]
Yes i do have proxy info with me but wasn't sure where to configure in this repo.
I did found some suggestions on this page:
https://www.quicklisp.org/beta/faq.html

But couldn't figure out which *.lisp to edit.
Closest reference I found was your circleci file which had sbf steps to install quicklist.

**Comment by tdrhq** [comment]
I can add a way to configure it, can you confirm what the http proxy looks like? Is it something like http://hostname:port? (I don't need the exact information, just wanted to make sure it's what I think it is as opposed to a SOCKS proxy or something similar)

**Comment by kaustubhshete** [comment]
yes its in this format: http://<proxyhost>:<proxyport>

**Comment by tdrhq** [comment]
@kaustubhshete Fixed in https://github.com/screenshotbot/screenshotbot-oss/commit/899f61a70f9ac48bb4449fa5543c4b2d26c70716

I haven't tested this against a real proxy, I tested it by setting an environment variable in the docker-compose.yml. In theory, you should be able to set it up in `~/.docker/config.json`, but that didn't really work for me. (I'm not a docker expert, I'm sure you're more familiar with this since you have to deal with your HTTP proxy on a regular basis :) )

Let me know if this works for you.

**Comment by kaustubhshete** [comment]
I already had proxy configured in ~/.docker/config.json.
So it worked when I added your changes.
Thanks a lot for quick fix on this.
